app/javascript/components/tab-bar.jsx

```diff
@@ -15,13 +15,17 @@ const progressBarFrames = keyframes`
   }
 `;
 
+const progressBarAnimation = props => css`
```

Comment:
why was this extracted from the ProgressBar definition?

Reply:
With v4, styled-components no longer supports interpolating keyframe declarations in untagged strings. 

Should I instead embed the css tag within the ProgressBar object? ex:
```
const ProgressBar = styled.div`
  background-color: ${colors.white};
  height: ${height}px;
  width: ${width}px;
  ${props => css`
    animation: ${progressBarFrames} ${props.time}s linear 0s;
    animation-fill-mode: forwards;
    animation-play-state: ${props.isPaused ? `paused` : `playing`};
  `};
`;
```

https://styled-components.com/docs/api#css

Reply:
no i think it's ok the way it is. Thanks for the reference!